### PR TITLE
Added 'default' marker

### LINKED ISSUE #136: RCOS "Starter/Default Projects"
Admins should be able to mark projects in RCOS as "Starter" or "Default", these projects are intended for users who just join RCOS to join.

There should be a button on the projects page "Mark Default"/"Unmark Default" that toggles a project's status.

### PR DESCRIPTION
**Pull request by zlalvani**
projects can now be marked as default, fixes #136

### CODE REVIEW

File: client/app/projects/profile/profile.controller.js [hunk 1]
```diff
@@ -4,6 +4,9 @@
 angular.module('observatory3App')
 .controller('ProjectsProfileCtrl', function ($scope, $http, Auth, $stateParams, Upload, Project, notify) {
     $scope.userOnProject = false;
+
+
```
Comment (agundy) [inline client/app/projects/profile/profile.controller.js]: a little too much added whitespace?

**Comment by agundy** [comment]
I think I missed something but what will this marked as default functionality be used for? Projects that people can be auto added to?

**Comment by zlalvani** [comment]
I think the idea is to let users join a project that has been marked as default when they register.

**Comment by zlalvani** [comment]
fixes #136

**Comment by seveibar** [comment]
@agundy this was part of our idea to prevent people from not having a project

**Pull request merged**